graph2tac/loader/py_data_server.py

```diff
@@ -1,5 +1,4 @@
 from pathlib import Path
-import capnp
```

Comment:
I can confirm @jasonrute's error. It seems to be caused by the removal of this line.

Reply:
ok, if it helps to put the line back, I can

Reply:
@mirefek Can you put it right above `import pytact.graph_api_capnp as graph_api_capnp` and make a comment that this line is needed for the next import to work?

Reply:
Linting tools will show it as a dead import, so it is good to have a comment to know why it is there.

Reply:
Yeah it looks like I was mistaken about the necessity. That being said, if you omit it you are implictly relying on other libraries to import it... That's somewhat fragile.